### DISCUSSION TITLE
Novos canais on line.

### DISCUSSION THREAD

**Discussion by cportugalr** [LITUATUI/M3UPT#45] [Ideas]
Poderia adicionar estes canais a sua lista m3u?
Existem outros canais regionais, eu cheguei a ter uma lista e um blog com os canais todos (o blog apaguei) a lista vou ver se encontro.
Existem mais canais portugueses.

Poderia criar um grupo no Facebook para sugestões de canais locais/regionais on line.

Rtp desporto
Zig zag

Kuriakos tv
Kuriakos kids
Kuriakos music
Kuriakos cine
Tv canção nova pt
Tv mana 1
Fatima tv
On fm tv 
Tvc (tv cabo verde)
Tpa
Ad buraca tv
Ad almada tv
Centro tv
Unife tv portugal 
Natacao tv

Web tv Paixão 

IURDTV Europa Ao Vivo

Sobrenatural TV Ao Vivo
Canal Vida Ao Vivo

ies TV - Igreja do Espírito Santo Ao Vivo
Igreja do Espírito Santo
Matosinhos tv,
Novum Canal,
ZTV,

TV MCJ,
TV SOCESSO,
SOICO TV (STV).
RADIO FALCLORE TV (PORTUGAL)
Rtv on

**Comment by LITUATUI** [comment]
Obrigado pela sugestão, mas não vou adicionar canais de propaganda religiosa a esta lista. Quanto aos regionais, depende da qualidade da emissão. Alguns só existem para passar publicidade e não acrescentam nada de útil.

Daqui para a frente planeio reduzir a quantidade de canais e não aumentar. Prefiro manter o foco na qualidade e não na quantidade.
